Src/BootCamp.Chapter/ArrayOps.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public static class ArrayOps
+    {
+        public static bool AreBalancesValid(string[] peopleAndBalance)
+        {
+            foreach (string field in peopleAndBalance)
+            {
+                string[] account = ConvertToAccountArray(field);
+                if (!Test.IsName(account[0]) || !Test.IsBalance(account[1..]))
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+
+        public static string[] ConvertToAccountArray(string personAndBalance)
+        {
+            string[] newArray;
+            try
+            {
+                newArray = personAndBalance.Split(Settings.stringSplitDivider);
+            }
+            catch (Exception)
+            {
+                throw new InvalidBalancesException();
+            }
+            return newArray;
+        }
+
+        public static decimal FindArrayMax(decimal[] inputArray)
```

Comment:
Array in name is redundant. Just do FindMax.

Reply:
I fixed it in [d1518fc](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/192/commits/d1518fc7884307a9ddc2f6074fa792682af6878d).